afids_utils/tests/test_transforms.py

```diff
@@ -101,10 +101,54 @@ def test_round_trip_world(
     def test_round_trip_voxel(
         self, afid_voxel: AfidVoxel, nii_affine: NDArray[np.float_]
     ):
-        afid_world = voxel_to_world(afid_voxel, nii_affine)
-        afid_voxel_approx = world_to_voxel(afid_world, nii_affine)
+        afid_world = af_xfm.voxel_to_world(afid_voxel, nii_affine)
+        afid_voxel_approx = af_xfm.world_to_voxel(afid_world, nii_affine)
 
         # Check to see if round-trip approximates to within 2 voxels
         assert afid_voxel_approx.i == pytest.approx(afid_voxel.i, abs=2)
         assert afid_voxel_approx.j == pytest.approx(afid_voxel.j, abs=2)
         assert afid_voxel_approx.k == pytest.approx(afid_voxel.k, abs=2)
+
+
+class TestCoordSystemXfm:
+    @given(afid_set=af_st.afid_sets())
+    @settings(
+        deadline=400,
+        suppress_health_check=[HealthCheck.function_scoped_fixture],
+    )
+    def test_invalid_new_coord_system(self, afid_set: AfidSet):
+        with pytest.raises(
+            ValueError, match=r"Unrecognized coordinate system.*"
+        ):
+            af_xfm.coord_system_xfm(afid_set, new_coord_system="invalid")
+
+    @given(afid_set=af_st.afid_sets(randomize_header=False))
+    @settings(
+        deadline=400,
+        suppress_health_check=[HealthCheck.function_scoped_fixture],
+    )
+    def test_same_coord_system(self, afid_set: AfidSet):
+        with pytest.raises(ValueError, match=r"Already saved in.*"):
+            af_xfm.coord_system_xfm(afid_set, new_coord_system="LPS")
+
+    @given(afid_set=af_st.afid_sets(randomize_header=False))
+    @settings(
+        deadline=400,
+        suppress_health_check=[HealthCheck.function_scoped_fixture],
+    )
+    def test_valid_new_coord_system(self, afid_set):
+        new_afid_set = af_xfm.coord_system_xfm(afid_set)
+
+        assert isinstance(new_afid_set, AfidSet)
+        assert new_afid_set.coord_system == "RAS"
+
+        for idx in range(len(new_afid_set.afids)):
```

Comment:
I think this would be a little more readable if the iterator line was `for old_afid, new_afid in zip(afid_set.afids, new_afid_set.afids):`